src/routes.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import { SideBar, Home, AboutMe, Resume, ContactMe, NotFound } from './sections/index';
```

Comment:
You should be able to import this from sections directly without specifying /index